crates/bevy_pbr/src/light/ambient_light.rs

```diff
@@ -15,8 +17,9 @@ use super::*;
 ///    ambient_light.brightness = 100.0;
 /// }
 /// ```
-#[derive(Resource, Clone, Debug, ExtractResource, Reflect)]
-#[reflect(Resource, Debug, Default)]
+#[derive(Resource, Component, Clone, Debug, ExtractResource, ExtractComponent, Reflect)]
+#[reflect(Resource, Component, Debug, Default)]
+#[require(Camera)]
```

Comment:
Not sure I like requiring `Camera` here, but not a strong enough opinion to block